presenter/src/main/java/com/mashup/chinchin/presenter/edit_preference/EditPreferenceActivity.kt

```diff
@@ -0,0 +1,148 @@
+package com.mashup.chinchin.presenter.edit_preference
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.mashup.chinchin.presenter.common.ChinChinQuestionCardState
+import com.mashup.chinchin.presenter.common.model.QuestionUiModel
+import com.mashup.chinchin.presenter.ui.common.*
+
+//TODO 27Line 데이터가 없으면 해당 액티비티가 실행되지 않아야 한다고 생각해서 이렇게 리턴했는데 의견 부탁드립니다.
+class EditPreferenceActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            val passedExtras: Bundle = intent.extras ?: return@setContent
```

Comment:
우리가 개발할 때 빈공간을 만들지 않는다면 null이 나오는 케이스는 없겠지만..
이렇게 리턴을 하게되면 빈 액티비티가 떠있는 모습이 되지 않을까??

현재처럼 setContent를 return 하는것보다는 차라리 exception을 발생시키는게 나을 수도 있어보이네

```suggestion
        setContent {
            val passedExtras: Bundle? = intent.extras
            checkNotNull(passedExtras)
```

이렇게하면 스마트캐스팅도 지원하고 오류 발생인 것을 더 명확히 알 수 있을거같아

Reply:
왓 코드까지 작성해주공 감동>~<